app/src/pages/_app.tsx

```diff
@@ -89,12 +91,33 @@ function App({ Component, pageProps }: AppProps) {
                       </Button>
                     </Link>
                   </Navbar.Section>
-
                   <Navbar.Section grow pt={18}>
                     <ServiceResourceButtons />
                   </Navbar.Section>
                 </>
               )}
+              {router.pathname.startsWith('/review') && (
+                <>
+                  <Stack>
+                    <Navbar.Section pt={18}>
+                      <Center>
+                        <Text c="gray">Measure Repository Artifacts</Text>
+                      </Center>
+                      <Divider my="md" />
+                      <Navbar.Section grow>
+                        <ServiceResourceButtons />
+                      </Navbar.Section>
+                    </Navbar.Section>
+                    <Navbar.Section grow pt={18}>
+                      <Center>
+                        <Text c="gray">Draft Artifacts</Text>
+                      </Center>
+                      <Divider my="md" />
+                    </Navbar.Section>
+                    <DraftResourceButtons />
```

Comment:
The `DraftResourceButtons` and `ServiceResourceButtons` components both call the `SidebarButtonList` component, which routes the resource buttons to certain paths. With the way things are set up right now, when I am on the "review" page and click on one of the resource buttons, I am routed to either the "repository" or "authoring" mode, and this gets reflected in the header. Not sure if this was intended, but I would expect that when you click on one of the resource buttons, you stay on the review mode. 

Wanted to see if this was discussed yet and whether it makes sense to make changes in this PR or in a future PR when the review page is further implemented. If we decide to make the change in this PR, I imagine that means adding some sort of conditional statement for what route prefix to use

Reply:
Definitely a great idea! I implemented it so that when a user clicks a side button in the side bar in the review mode, they will remain in the review mode and see a list that is identical to what they would see in the authoring/repository mode. However when viewing the resource list in the review mode, the user can still be routed to the repository/authoring mode if they click a specific button on the card resource (i.e The draft resource edit button still routes to the authoring mode). Curious to know your thoughts about how it looks/ if it makes sense those buttons would still route the user to other modes.